README.md

```diff
@@ -1,82 +1,81 @@
-# My Awesome Step
+# Export macOS Xcode archive
 
-My Awesome Step is a solid starting code base for
-a new Step.
+[![Step changelog](https://shields.io/github/v/release/bitrise-steplib/steps-export-xcarchive-mac?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/steps-export-xcarchive-mac/releases)
 
+Export macOS Xcode archive
 
-## How to use this Step
+<details>
+<summary>Description</summary>
 
-Can be run directly with the [bitrise CLI](https://github.com/bitrise-io/bitrise),
-just `git clone` this repository, `cd` into it's folder in your Terminal/Command Line
-and call `bitrise run test`.
+Export macOS Xcode archive.
```

Comment:
Maybe expand on the description a bit and just have one sentence

Reply:
@shams-ahmed how about now?

Reply:
looks good!